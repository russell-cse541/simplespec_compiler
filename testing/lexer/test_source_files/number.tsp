-0n 0n -0i 0i -0 0
-1122 -88392    -21 -3
1 2 3 4 5 6 7 8 9 10 ^
1n 2n 3n 4n 5n 6n 7n 8n 9n 10n
1i 2i 3i 4i 5i 6i 7i 8i 9i 10i
-1n -2n -3n -4n -5n -6n -7n -8n -9n -10n
-1i -2i -3i -4i -5i -6i -7i -8i -9i -10i
-1 -2 -3 -4 -5 -6 -7 -8 -9 -10
-1234567890n $ -1234567890i -1234567890
1234567890n 1234567890i 1234567890
