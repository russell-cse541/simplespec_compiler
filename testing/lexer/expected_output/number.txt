
-0n                    TagNum: 306    Line: 1     Col: 1          
0n                     TagNum: 306    Line: 1     Col: 5          
-0i                    TagNum: 307    Line: 1     Col: 8          
0i                     TagNum: 307    Line: 1     Col: 12         
-0                     TagNum: 308    Line: 1     Col: 15         
0                      TagNum: 308    Line: 1     Col: 18         
-1122                  TagNum: 308    Line: 2     Col: 1          
-88392                 TagNum: 308    Line: 2     Col: 7          
-21                    TagNum: 308    Line: 2     Col: 17         
-3                     TagNum: 308    Line: 2     Col: 21         
1                      TagNum: 308    Line: 3     Col: 1          
2                      TagNum: 308    Line: 3     Col: 3          
3                      TagNum: 308    Line: 3     Col: 5          
4                      TagNum: 308    Line: 3     Col: 7          
5                      TagNum: 308    Line: 3     Col: 9          
6                      TagNum: 308    Line: 3     Col: 11         
7                      TagNum: 308    Line: 3     Col: 13         
8                      TagNum: 308    Line: 3     Col: 15         
9                      TagNum: 308    Line: 3     Col: 17         
10                     TagNum: 308    Line: 3     Col: 19         

Unrecognized Token Line : 3 Column : 22 


1n                     TagNum: 306    Line: 4     Col: 1          
2n                     TagNum: 306    Line: 4     Col: 4          
3n                     TagNum: 306    Line: 4     Col: 7          
4n                     TagNum: 306    Line: 4     Col: 10         
5n                     TagNum: 306    Line: 4     Col: 13         
6n                     TagNum: 306    Line: 4     Col: 16         
7n                     TagNum: 306    Line: 4     Col: 19         
8n                     TagNum: 306    Line: 4     Col: 22         
9n                     TagNum: 306    Line: 4     Col: 25         
10n                    TagNum: 306    Line: 4     Col: 28         
1i                     TagNum: 307    Line: 5     Col: 1          
2i                     TagNum: 307    Line: 5     Col: 4          
3i                     TagNum: 307    Line: 5     Col: 7          
4i                     TagNum: 307    Line: 5     Col: 10         
5i                     TagNum: 307    Line: 5     Col: 13         
6i                     TagNum: 307    Line: 5     Col: 16         
7i                     TagNum: 307    Line: 5     Col: 19         
8i                     TagNum: 307    Line: 5     Col: 22         
9i                     TagNum: 307    Line: 5     Col: 25         
10i                    TagNum: 307    Line: 5     Col: 28         
-1n                    TagNum: 306    Line: 6     Col: 1          
-2n                    TagNum: 306    Line: 6     Col: 5          
-3n                    TagNum: 306    Line: 6     Col: 9          
-4n                    TagNum: 306    Line: 6     Col: 13         
-5n                    TagNum: 306    Line: 6     Col: 17         
-6n                    TagNum: 306    Line: 6     Col: 21         
-7n                    TagNum: 306    Line: 6     Col: 25         
-8n                    TagNum: 306    Line: 6     Col: 29         
-9n                    TagNum: 306    Line: 6     Col: 33         
-10n                   TagNum: 306    Line: 6     Col: 37         
-1i                    TagNum: 307    Line: 7     Col: 1          
-2i                    TagNum: 307    Line: 7     Col: 5          
-3i                    TagNum: 307    Line: 7     Col: 9          
-4i                    TagNum: 307    Line: 7     Col: 13         
-5i                    TagNum: 307    Line: 7     Col: 17         
-6i                    TagNum: 307    Line: 7     Col: 21         
-7i                    TagNum: 307    Line: 7     Col: 25         
-8i                    TagNum: 307    Line: 7     Col: 29         
-9i                    TagNum: 307    Line: 7     Col: 33         
-10i                   TagNum: 307    Line: 7     Col: 37         
-1                     TagNum: 308    Line: 8     Col: 1          
-2                     TagNum: 308    Line: 8     Col: 4          
-3                     TagNum: 308    Line: 8     Col: 7          
-4                     TagNum: 308    Line: 8     Col: 10         
-5                     TagNum: 308    Line: 8     Col: 13         
-6                     TagNum: 308    Line: 8     Col: 16         
-7                     TagNum: 308    Line: 8     Col: 19         
-8                     TagNum: 308    Line: 8     Col: 22         
-9                     TagNum: 308    Line: 8     Col: 25         
-10                    TagNum: 308    Line: 8     Col: 28         
-1234567890n           TagNum: 306    Line: 9     Col: 1          

Unrecognized Token Line : 9 Column : 14 


-1234567890i           TagNum: 307    Line: 9     Col: 16         
-1234567890            TagNum: 308    Line: 9     Col: 29         
1234567890n            TagNum: 306    Line: 10    Col: 1          
1234567890i            TagNum: 307    Line: 10    Col: 13         
1234567890             TagNum: 308    Line: 10    Col: 25         

Below is a list of errors (500 max)
-----------------------------------
Unrecognized Token On line 3 column 22
Unrecognized Token On line 9 column 14


Token Count : 76
